spinta/types/model.py

```diff
@@ -210,11 +224,8 @@ def load(
     prop.lang = load_lang_data(context, prop.lang)
     prop.comments = load_comments(prop, prop.comments)
 
-    # Parse dtype like geometry(point, 3346)
     if data['type'] is None:
         raise UnknownPropertyType(prop, type=data['type'])
-    dtype_type, dtype_args = _parse_dtype_string(data['type'])
```

Comment:
If this function is not used here anymore, then it should be moved to tabular helpers.